app/src/main/java/com/edu/movie/screen/commonview/adapter/ItemRecyclerViewGrid.kt

```diff
@@ -0,0 +1,27 @@
+package com.edu.movie.screen.commonview.adapter
+
+import android.os.AsyncTask
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import com.edu.movie.R
+import com.edu.movie.data.model.MovieItem
+import com.edu.movie.utils.Constant
+import com.edu.movie.utils.LoadImageFromUrl
+import kotlinx.android.synthetic.main.item_movie_gridlayout.view.*
+
+class MovieGridViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
```

Comment:
đổi lại tên class. tên file và class phải giống với nhau
```suggestion
class ItemMovieGridViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
```